pkg/config/validation/validation_test.go

```diff
@@ -2127,6 +2127,59 @@ func TestValidateHTTPRewrite(t *testing.T) {
 	}
 }
 
+func TestValidateUriRegexRewrite(t *testing.T) {
+	testCases := []struct {
+		name  string
+		in    *networking.RegexRewrite
+		valid bool
+	}{
+		{
+			name:  "uriRegexRewrite nil",
+			in:    nil,
+			valid: true,
+		},
+		{
+			name: "uriRegexRewrite happy path",
+			in: &networking.RegexRewrite{
+				Match:   "^/service/([^/]+)(/.*)$",
+				Rewrite: "\\2/instance/\\1",
```

Comment:
```suggestion
				Rewrite: `\2/instance/\1`,
```

If you don't want to worry about escaping